tests/sentry/eventstream/kafka/test_consumer_strategy.py

```diff
@@ -32,7 +32,7 @@ def get_kafka_payload() -> KafkaPayload:
                     "skip_consume": False,
                 },
             ]
-        ),
```

Comment:
Needed?

Reply:
yep, otherwise it complained about getting `str` here where it's annotated `bytes` in arroyo